frontend/app/src/entities/proposed-changes/api/updateReviewFromApi.ts

```diff
@@ -0,0 +1,36 @@
+import graphqlClient from "@/shared/api/graphql/graphqlClientApollo";
+import { BranchContextParams } from "@/shared/api/types";
+import { gql } from "@apollo/client";
+
+const MUTATION = `
+mutation ProposedChangeReview($id: String!, $decision: ProposedChangeApprovalDecision!) {
+  CoreProposedChangeReview(data: {
+    id: $id,
+    decision: $decision
+  }) {
+    ok
+  }
+}
+`;
+
+export interface UpdateReviewFromApiApiParams extends BranchContextParams {
+  proposedChangeId: string;
+  decision: string;
+}
+
+export function udpateReviewFromApi({
```

Comment:
- There is a typo `udpate` 
- I suggest to add more context in the name of the function: Review -> ProposedChangeReview